docs/developer/payments/transactions.mdx

```diff
@@ -407,22 +410,26 @@ Query variables:
 }
 ```
 
-:::info
-Enabling automatic checkout completion will not disrupt storefront functionality.
-If `checkoutComplete` is called for a `Checkout` that has already been converted to an `Order`,
-the corresponding `Order` instance will be return.
+:::danger
+**One this setting is enabled, you should verify if your connected services, including storefront, can handle this flow.**
+
+Please remember, that if setting is enabled:
+- `Checkout` will be deleted after checkout completion, which means any query or mutation relying on it will no longer work after that
```

Comment:
```suggestion
- `Checkout` will be deleted after checkout completion which means that any query or mutation relying on it will no longer work after that
```